.zenodo.json

```diff
@@ -282,16 +290,14 @@
       "name": "Thirion, Bertrand",
       "orcid": "0000-0001-5018-7895"
     },
+    {
+      "name": "Gauthier, Carl"
```

Comment:
@krallznut I've added you to the Zenodo fragment, and you will be listed as an author in the [Zenodo archive](https://doi.org/10.5281/zenodo.591597). Please let me know if you would like your name to be rendered otherwise, or if you have an affiliation and/or [ORCID](https://orcid.org/) you would to use. I can also remove you if you'd rather not be listed as an author.